tasks/extract_filepaths_for_granule/schemas/config.json

```diff
@@ -0,0 +1,55 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "extract_filepaths_for_granule Lambda Config",
+    "description": "The config for the extract_filepaths_for_granule Lambda function.",
+    "type": "object",
+    "properties": {
+      "glacier-bucket": {
```

Comment:
Per our discussion, the *-bucket keys should be replaced by a single key that pulls in all the bucket configurations.